lambdas/indexer/.chalice/config.json.template.py

```diff
@@ -24,10 +24,14 @@
                 },
                 "write": {
                     "reserved_concurrency": config.indexer_concurrency,
-                    # FIXME: Use a two-stage pipeline with a high memory size 2nd
-                    #        stage to process tallies that failed in the lower
-                    #        size 1st stage. https://github.com/DataBiosphere/azul/issues/1732
-                    "lambda_memory_size": 3008,
+                    "lambda_memory_size": 256,
+                    # TODO: Consider lowering the lambda timeout and write queue
+                    # visibility to speed up big, doomed to fail aggregations
```

Comment:
Should this be accompanied with a ticket link?

Reply:
Yea, I kinda wanted to leave it in so that when Hannes reviews the PR he can decide what to do here.